python/samples/kernel-syntax-examples/chat_gpt_api.py

```diff
@@ -25,10 +25,11 @@
 
 api_key, org_id = sk.openai_settings_from_dot_env()
 kernel.add_service(
-    sk_oai.OpenAIChatCompletion(service_id="chat-gpt", ai_model_id="gpt-3.5-turbo", api_key=api_key, org_id=org_id)
+    sk_oai.OpenAIChatCompletion(
+        service_id="chat-gpt", ai_model_id="gpt-3.5-turbo", api_key=api_key)
 )
-
-settings = kernel.get_prompt_execution_settings_from_service(ChatCompletionClientBase, "chat-gpt")
+service_id = "chat-gpt"
+settings = kernel.get_prompt_execution_settings_from_service_id("chat-gpt", ChatCompletionClientBase)
```

Comment:
why not use the service_id variable you just created, and maybe move that to the line 26, and use it there as well!

Reply:
okay sure

Reply:
am doing some changes into code to make it bit clean